common/src/main/java/com/viaversion/viaversion/protocols/protocol1_13to1_12_2/blockconnections/ConnectionData.java

```diff
@@ -693,4 +619,89 @@ interface ConnectorInitAction {
+            List<BlockChangeRecord1_8> updates = new ArrayList<>();
+            for (int chunkDeltaX = -1; chunkDeltaX <= 1; chunkDeltaX++) {
+                for (int chunkDeltaZ = -1; chunkDeltaZ <= 1; chunkDeltaZ++) {
+                    int distance = Math.abs(chunkDeltaX) + Math.abs(chunkDeltaZ);
+                    if (distance == 0) continue;
+
+                    int chunkMinX = (chunkX + chunkDeltaX) << 4;
+                    int chunkMinZ = (chunkZ + chunkDeltaZ) << 4;
+                    if (distance == 2) { // Corner
+                        for (int blockY = chunkMinY; blockY < chunkMinY + 16; blockY++) {
+                            int blockPosX = chunkDeltaX == 1 ? 0 : 15;
+                            int blockPosZ = chunkDeltaZ == 1 ? 0 : 15;
+                            updateBlock(chunkMinX + blockPosX, blockY, chunkMinZ + blockPosZ, updates);
+                        }
+                    } else {
+                        for (int blockY = chunkMinY; blockY < chunkMinY + 16; blockY++) {
+                            int xStart, xEnd;
+                            int zStart, zEnd;
+                            if (chunkDeltaX == 1) {
+                                xStart = 0;
+                                xEnd = 2;
+                                zStart = 0;
+                                zEnd = 16;
+                            } else if (chunkDeltaX == -1) {
+                                xStart = 14;
+                                xEnd = 16;
+                                zStart = 0;
+                                zEnd = 16;
+                            } else if (chunkDeltaZ == 1) {
+                                xStart = 0;
+                                xEnd = 16;
+                                zStart = 0;
+                                zEnd = 2;
+                            } else {
+                                xStart = 0;
+                                xEnd = 16;
+                                zStart = 14;
+                                zEnd = 16;
+                            }
+                            for (int blockX = xStart; blockX < xEnd; blockX++) {
+                                for (int blockZ = zStart; blockZ < zEnd; blockZ++) {
+                                    updateBlock(chunkMinX + blockX, blockY, chunkMinZ + blockZ, updates);
+                                }
+                            }
+                        }
+                    }
+
+                    if (!updates.isEmpty()) {
+                        PacketWrapper wrapper = PacketWrapper.create(ClientboundPackets1_13.MULTI_BLOCK_CHANGE, null, user);
+                        wrapper.write(Type.INT, chunkX + chunkDeltaX);
+                        wrapper.write(Type.INT, chunkZ + chunkDeltaZ);
+                        wrapper.write(Type.BLOCK_CHANGE_RECORD_ARRAY, updates.toArray(EMPTY_RECORDS));
+                        try {
+                            wrapper.send(Protocol1_13To1_12_2.class);
```

Comment:
Although from the previous code, please make the method throw Exception and remove the try catch here